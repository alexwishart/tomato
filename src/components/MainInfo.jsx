import React from "react";
import styled from 'styled-components'
import tomatoImage from "../images/tomato.jpg";

const MainInfo = () => {
    return <>
    <h1>The Tomato</h1>
      <TomatoImage src={tomatoImage} />
      </>
}

const TomatoImage = styled.img`
  width: 250px;  
`

export default MainInfo;