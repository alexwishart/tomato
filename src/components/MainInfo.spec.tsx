import React from 'react';
import { render } from '@testing-library/react';
import MainInfo from './MainInfo';

describe('The MainInfo component', () => {
  it('should render correctly', () => {
    expect(render(<MainInfo />)).toMatchSnapshot();
  });
});
