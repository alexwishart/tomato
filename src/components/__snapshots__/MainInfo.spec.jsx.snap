// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The MainInfo component should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bdfBwQ bBcGKn"
      >
        <img
          alt="Image of tomato"
          class="sc-gsTCUz fBVUqk"
          src="tomato.jpg"
        />
        <h2>
          Varieties
        </h2>
        <p>
          There are around 7500 tomato varieties grown for various purposes. Heirloom tomatoes are becoming increasingly popular, particularly among home gardeners and organic producers, since they tend to produce more interesting and flavorful crops at the cost of disease resistance and productivity. In 1973, Israeli scientists developed the world's first long shelf-life commercial tomato varieties.
        </p>
        <p>
          Tomato varieties are roughly divided into several categories, based mostly on shape and size.
        </p>
        <ul>
          <li>
            "Slicing"/"globe" tomatoes
          </li>
          <li>
            Beefsteak tomatoes
          </li>
          <li>
            Oxheart tomatoes
          </li>
          <li>
            Plum tomatoes
          </li>
          <li>
            Pear tomatoes
          </li>
          <li>
            Cherry tomatoes
          </li>
          <li>
            Grape tomatoes
          </li>
          <li>
            Campari tomatoes
          </li>
        </ul>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bdfBwQ bBcGKn"
    >
      <img
        alt="Image of tomato"
        class="sc-gsTCUz fBVUqk"
        src="tomato.jpg"
      />
      <h2>
        Varieties
      </h2>
      <p>
        There are around 7500 tomato varieties grown for various purposes. Heirloom tomatoes are becoming increasingly popular, particularly among home gardeners and organic producers, since they tend to produce more interesting and flavorful crops at the cost of disease resistance and productivity. In 1973, Israeli scientists developed the world's first long shelf-life commercial tomato varieties.
      </p>
      <p>
        Tomato varieties are roughly divided into several categories, based mostly on shape and size.
      </p>
      <ul>
        <li>
          "Slicing"/"globe" tomatoes
        </li>
        <li>
          Beefsteak tomatoes
        </li>
        <li>
          Oxheart tomatoes
        </li>
        <li>
          Plum tomatoes
        </li>
        <li>
          Pear tomatoes
        </li>
        <li>
          Cherry tomatoes
        </li>
        <li>
          Grape tomatoes
        </li>
        <li>
          Campari tomatoes
        </li>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
